format/format.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2022 Scorecard Authors
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package format
```

Comment:
Maybe move to `pkg` which already has bunch of exposed helper fns to output ScorecardResult.

Reply:
Will check it out!